databricks/koalas/frame.py

```diff
@@ -3077,14 +3077,12 @@ def between_time(
         2018-04-09 00:00:00  1
         2018-04-12 01:00:00  4
         """
-        from databricks.koalas.indexes import DatetimeIndex
-
         axis = validate_axis(axis)
 
         if axis != 0:
             raise NotImplementedError("between_time currently only works for axis=0")
 
-        if not isinstance(self.index, DatetimeIndex):
+        if not isinstance(self.index, ks.DatetimeIndex):
```

Comment:
Could you also update it in `at_time`?

Reply:
> @awdavidson Thanks for working on this!
> I left some comments. Could you address them? Thanks!

No problem. Comments have been address :)